### DISCUSSION TITLE
Transparent taskbar

### DISCUSSION THREAD

**Discussion by Ropicek** [valinet/ExplorerPatcher#2976] [Q&A]
After the update KB5035853 (Win11) I have a transparent taskbar. My ExplorerPatcher version is 22621.2681.62.2. My taskbar style is Windows 10. Can you help me, please? I'd like to have a black taskbar. Thanks.
![taskbar](https://github.com/valinet/ExplorerPatcher/assets/163191981/6a1ad944-b1c2-443c-8f0a-22777cca293d)

**Comment by ghost** [comment]
I have the exact same problem. EP was unable to download symbols probably because the Win 11 build is too recent. 
I'm on the way to flush (EP) too unstable

**Comment by johnkarlhill** [comment]
I have the same issue on 2 different laptops.  Both updated today to latest build and taskbar is transparent.  Installing TranslucentTB will overlay it to make it less annoying, but I would prefer to not have yet another TSR app. 
![Screenshot 2024-03-12 150007](https://github.com/valinet/ExplorerPatcher/assets/112784549/de4c89a0-820b-4bd4-ac33-4c92c6d909d1)

**Reply by TKASAKAWA** [reply]
I agree
I don't want to modify the registry or change it with other apps

**Comment by RaptorLords** [comment]
Same problem as you, hopefully fix soon. It is making my wallpaper engine laggy as well without the taskbar.

**Comment by jimsinenomine** [comment]
As a workaround, you can always uninstall the Windows Updates and pause them until a solution is provided.  That's what I did for now.

**Reply by RaptorLords** [reply]
This is what I did for the time being and paused updates since we don’t have an eta on a fix yet.

**Reply by ghost** [reply]
Problem is not Windows updates, it's ExplorerPatcher not able to cope with Win updates.
I personally uninstall EP on all my computers.
EP has the reputation to be unstable and may cause bigger problems than that (explorer crashing, windows not booting)

**Comment by TKASAKAWA** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/147231609/214c492d-5c8b-40d6-8dca-85f2ee3b807d)
We are also experiencing the same symptoms
Version 22621.3007.63.3
Windows 11, OS build 22631.3296.

**Comment by TKASAKAWA** [comment]
How to temporarily fix it
Case of Using ExplorerPatcher+Open-Shell
![image](https://github.com/valinet/ExplorerPatcher/assets/147231609/9efc5ff5-8ed9-49d5-84e4-cf9852fc5b64)

**Comment by TKASAKAWA** [comment]
> After the update KB5035853 (Win11) I have a transparent taskbar. My ExplorerPatcher version is 22621.2681.62.2. My taskbar style is Windows 10. Can you help me, please? I'd like to have a black taskbar. Thanks. ![taskbar](https://private-user-images.githubusercontent.com/163191981/312212322-6a1ad944-b1c2-443c-8f0a-22777cca293d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAzMDg0MTEsIm5iZiI6MTcxMDMwODExMSwicGF0aCI6Ii8xNjMxOTE5ODEvMzEyMjEyMzIyLTZhMWFkOTQ0LWIxYzItNDQzYy04ZjBhLTIyNzc3Y2NhMjkzZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxM1QwNTM1MTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01NjQ5ZmE4YTdhZTg0OWQ0YmM4ZTlkOWJlOTM4ODMwMmI4NjczOTk0OTZlZTNiNTk5ZGRlZDZjMjQxMTcyN2FkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-HpcNzX3mWxLPYIqKB0uz1yWx-K8efl1Y7bmkkHBzL0)

![image](https://github.com/valinet/ExplorerPatcher/assets/147231609/7ac1d695-2235-4c53-959d-42eb7ccfc490)

**Comment by RaptorLords** [comment]
Being patched in 24 hours

**Comment by jimsinenomine** [comment]
Fixed now with 22621.3296.64.1.  Working fine on my side.  FYI

**Comment by johnkarlhill** [comment]
Worked here as well.  Fantastic job!
